### PR TITLE
Unfold

### PR DESCRIPTION
**Pull request by sdenaro**
for https://github.com/ampproject/amp.dev/issues/4970

### CODE REVIEW

**Comment by sdenaro** [comment]
@sebastianbenz  @shishirm Is there a reason why this is being held up?

**Comment by sebastianbenz** [comment]
I was waiting for @shishirm 's approval

**Review by shishirm — APPROVED** [review]

**Review by shishirm — APPROVED** [review]
LGTM

**Review by sebastianbenz — APPROVED** [review]

**Pull request merged**
